app/assets/stylesheets/components/common/_global_nav.scss

```diff
@@ -177,7 +193,7 @@
   @include respond-to($mq-m) {
     display: block;
     padding: $baseline-unit*2 $baseline-unit;
-    font-weight: 700;
+    
```

Comment:
Can we keep this font weight on desktop? (was part of the last visual improvement, but I think the lighter font is better on mobile).